### DISCUSSION TITLE
Not honouring conversion settings.

### DISCUSSION THREAD

**Discussion by PetrusVermaak** [yermak/AudioBookConverter#436] [General]
I have an audiobook with chapters in m4b format as single file.

I want the software to reconvert to 16kb/s and 44.1kHz duel channel. It ends up splitting into chapters, and reconverts to 128kbs even though I tried to save my conversion setting as default.

How do I resolve this? I'm running Windows.

**Comment by yermak** [comment]
Should be fixed in version 6.2.6
